scripts/cargo-rmc

```diff
@@ -25,6 +26,8 @@ def main():
     parser.add_argument("--mangler", default="v0")
     parser.add_argument("--visualize", action="store_true")
     parser.add_argument("--srcdir", default=".")
+    parser.add_argument("--target-dir", default="target",
```

Comment:
I don't know how to comment on a non-changed file, but should we be adding this argument to `scripts/rmc` as well?

Reply:
I thought about that, but I could not come to a conclusion. Normally, when someone runs `cargo`, they expect all artifacts (besides `Cargo.lock`) to be in the target directory. `rustc` has a somewhat similar option `--out-dir` for specifying the directory of the compiled file. However, it's not really treated as a build directory and I don't know if we should treat it as such. So, I think this is a decision for the team to make.

Reply:
In addition to @bdalrhm explanation, I would like to point out that RMC's workflow is more similar to CBMC's: It does generate a verification output, not a compiled program. So it makes sense for us to have this `--target-dir` flag for `cargo-rmc` but not for `rmc` itself.